src/discord-cluster-manager/migrations/20241208_01_p3yuR-initial-leaderboard-schema.py

```diff
@@ -0,0 +1,46 @@
+"""
```

Comment:
>how come this is dated?

I believe it's dated just as a kind of sequence number. I actually prefer numbering migrations in increasing order without gaps. Some of the yoyo documentation has that style; however `yoyo new` uses dates, and I thought it would be less confusing to stick with the dates rather than change the generated file name.

>is the idea that we version the schema using these python files?

Yes, that's exactly the idea. Versioning the schema makes the process of rolling the version forward less complex, and it provides a useful view of recent changes to the schema.